qutebrowser/browser/webengine/interceptor.py

```diff
@@ -33,6 +33,50 @@ class RequestInterceptor(QWebEngineUrlRequestInterceptor):
 
     """Handle ad blocking and custom headers."""
 
+    # A simple mapping from QWebEngine Resource Types to qutebrowser extension
+    # ResourceTypes. If a ResourceType is added to Qt, this table should be
```

Comment:
You're using various different words for the same concept here (map, mapping, table) - but none of them is what Python usually calls it (a dictionary) :wink:

I'm fine with "A simple mapping", but I'd prefer "this dict should be" :wink: